odk1-src/form-question-types.rst

```diff
@@ -805,6 +805,43 @@ See :ref:`image-options` to learn more about including images in surveys.
   abcd_icon,d,D,d.jpg
 
 
+.. _likert-image-widget:
+
+Select widget with likert
+""""""""""""""""""""""""""""""""""
+type
+ :tc:`select_one {list_name}`
+appearance
+ :tc:`likert`
+
+Select widget supports likert options. Options can include text, images or both. If both are provided, images appear above text.
+
+Take into consideration that likert options DO NOT wrap to next line.
```

Comment:
It would be helpful to clarify the implications of this. Maybe something like "Take into consideration that likert options DO NOT wrap to next line and will overlap instead. If you have long option text, make sure to try the question on the device you will use for data collection."

Reply:
It doesn't look like the update for this got committed, @shouryaj!

Reply:
My bad! Not sure what happened there. Just pushed this change up again.